capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative './decorator'
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.upcase
```

Comment:
- [ ] It will be great to actually capitalize the first letter. Instead of `upcase` it should be `capitalize`. The difference between these two is that the upcase will actually make all the letters in the word capital letters but capitalize will only make the first letter a capital letter. The correct implementation should be capitalize

Reply:
This change has been done well